src/features/app/layouts/MainLayout/MainLayout.tsx

```diff
@@ -4,7 +4,12 @@ import Navbar from '../../components/Navbar';
 
 import { MainContainer } from './MainLayout.styles';
 
-const MainLayout = ({ children, noHeader }) => (
+interface MainLayoutProps {
+  children: React.ReactNode;
+  noHeader: boolean;
+}
+
+const MainLayout = ({ children, noHeader }: MainLayoutProps) => (
```

Comment:
Do you have to explicitly define the return of your component or is kind of redundant?